packages/esm-patient-search-app/src/patient-search/patient-search.component.tsx

```diff
@@ -25,6 +25,8 @@ interface PatientSearchProps {
 }
 
 const PatientSearch: React.FC<PatientSearchProps> = ({ hidePanel }) => {
+  const layout = useLayoutType();
+  const isTablet = React.useMemo(() => layout === 'tablet', [layout]);
```

Comment:
For me the whole search layout is broken in desktop. Not sure why a special case for tablet can help us.

Also, why don't we tackle this problem in CSS? Why do we need to distinguish between these cases in JS?